morpheus/_lib/include/morpheus/stages/preprocess_nlp.hpp

```diff
@@ -89,6 +101,8 @@ class PreprocessNLPStage
                        int stride         = -1,
                        std::string column = "data");
 
+    std::shared_ptr<OutputT> pre_process_batch(std::shared_ptr<InputT> x);
```

Comment:
Looking at the other classes, lets standardize on `on_data` for the name of the function for each stage which handles messages.